src/bloqade/task/base.py

```diff
@@ -259,11 +328,20 @@ def rydberg_densities(
 
         Args:
             filter_perfect_filling (bool, optional): whether return will
-            only contain perfect filling shots. Defaults to True.
-
-        Return:
-            per-site rydberg density for each task as a pandas DataFrame or Series.
+                only contain perfect filling shots. Defaults to True.
+            clusters: (tuple[int, int], Sequence[Tuple[int, int]]):
+                cluster index to filter shots from. If none are provided
+                all clusters are used, defaults to [].
 
+        Returns:
+            rydberg_densities (Union[pd.Series, pd.DataFrame]):
+                per-site rydberg density for each task as a pandas DataFrame or Series.
+                For example:
+                ```python
+                0      1
+                task_number
+                0            0.053  0.054
```

Comment:
put this as a separate example? it may not render nicely for multi-line code blocks within the method signature table.

Reply:
Ok I think that's fine then.